Creating a markdown version

DKMonroe opened the pull request:
I thought you might appreciate a markdown version of your text, which will make it viewable on Github, and more likely to be used by Githubbers (or whatever they are called). Right now, I've only added spaces to your text and I'll spend a week or so adding markup. 

Could you send me your images? I'll make a folder and add them as links in the markup. I can do it without the images, but it would look sweet! Also, I'm sure you would like the markup version to look similar to the PDF version.

Thanks for writing this and sharing it on Github.

DKMonroe committed:
Create DataScienceBook.md

Just a copy/paste from the text. More to come.

DKMonroe committed:
Adding spacing up to line 365.

DKMonroe committed:
Added spaces up to line 614.

DKMonroe committed:
Finished adding spaces. Now for the markup.

jmstanto commented:
Hi David

Can you confirm that you received this message and send me your email address? I apologize that I did not get back to you sooner. Heavy travel schedule!

I can send you many of the images next week.

Great idea! Thanks,

Jeff

No Higgs Bosons were harmed in the transmission of this message.

On Feb 26, 2013, at 8:00 AM, "David Monroe" <notifications@github.com<mailto:notifications@github.com>> wrote:

I thought you might appreciate a markdown version of your text, which will make it viewable on Github, and more likely to be used by Githubbers (or whatever they are called). Right now, I've only added spaces to your text and I'll spend a week or so adding markup.

Could you send me your images? I'll make a folder and add them as links in the markup. I can do it without the images, but it would look sweet! Also, I'm sure you would like the markup version to look similar to the PDF version.

Thanks for writing this and sharing it on Github.

---

You can merge this Pull Request by running

  git pull https://github.com/DKMonroe/data-science-r master

Or view, comment on, or merge it at:

  https://github.com/jmstanto/data-science-r/pull/1

Commit Summary
-   Create DataScienceBook.md
-   Adding spacing up to line 365.
-   Added spaces up to line 614.
-   Finished adding spaces. Now for the markup.

File Changes
-   A DataScienceBook.mdhttps://github.com/jmstanto/data-science-r/pull/1/files#diff-0 (2566)

Patch Links:
-   https://github.com/jmstanto/data-science-r/pull/1.patch
-   https://github.com/jmstanto/data-science-r/pull/1.diff

DKMonroe commented:
Message received: five by five. :-)

I had to laugh when I read, "No Higgs Bosons were harmed in the transmission of this message."

DKMonroe committed:
Did the markeup up to Chapter 4.

DKMonroe committed:
Update DataScienceBook.md

DKMonroe committed:
Finished Chapter 4 and started 5. Checking the tables.

DKMonroe committed:
Finished Chapter 4 and started 5. Checking the tables.

DKMonroe committed:
Finished chapters 5 and 6. I still need to deal with tables.

DKMonroe committed:
Finished Chapter 7. Still need to work on tables.

DKMonroe commented:
My email is on my profile, but send the images to this one: dorje.kawabata@gmail.com. 

I'm working on the tables. They're a pain, but it's coming along. I'm almost halfway done with the markup.

DKMonroe committed:
Chapter 8 marked, but some code is being funny.

DKMonroe committed:
Fixing the source for chapter 7 to see if I can line it up properly.

DKMonroe committed:
Fixing the source sections.

DKMonroe committed:
recommit

I think this was committed a long time ago. Looks like I should push it
just in case.

DKMonroe commented:
I thought it might be helpful to have a markdown version available. Here is what I've done so far. If people want to add to it, that's great too. I'll add more when I get time.
